src/components/App.tsx

```diff
@@ -35,6 +35,8 @@ import {
   initStore,
   updateFileNameAndDescription,
   deleteFile,
+  addFilePreview,
+  removeFilePreview,
```

Comment:
I see how to open preview, can you explain how to close preview?

If there is not close preview action, we need to remove this import and action.